ipatests/prci_definitions/temp_commit.yaml

```diff
@@ -45,14 +45,38 @@ jobs:
         timeout: 1800
         topology: *build
 
-  fedora-29/temp_commit:
+  fedora-29/test_automount_locations:
     requires: [fedora-29/build]
     priority: 50
     job:
       class: RunPytest
       args:
         build_url: '{fedora-29/build_url}'
-        test_suite: test_integration/test_REPLACEME.py
+        test_suite: test_integration/test_automount_locations.py
```

Comment:
This test is probably a copy/paste error from another PR. Not a big deal as temp_commit will be removed, though.

Reply:
That was on purpose.
That test installs a server and a client with a non-modified mask and still work with this change.

Reply:
Hi @flo
Thanks for the comments, I've updated the PR to add the new tests to f28+f29+rawhide.